skt/state_file.py

```diff
@@ -57,26 +55,30 @@ def read(cfg):
             logging.error("Failed to read state file: %s", exception)
             raise exception
 
+    current_state['state_file'] = state_file
```

Comment:
How do we carry around the state_file then?  I agree with your assessment, I was trying to avoid the ugly alternative which is passing an array of state_files to each cmd.  So I squirreled it away in the state dict for now.